app/scenes/Collection/Collection.js

```diff
@@ -145,14 +155,13 @@ class CollectionScene extends Component {
               />{' '}
               {this.collection.name}
             </Heading>
+
+            {hasPinnedDocuments && <DocumentList documents={pinnedDocuments} />}
+
             <Subheading>Recently edited</Subheading>
-            <DocumentList
-              documents={this.props.documents.recentlyEditedIn(
```

Comment:
@jorilallo maybe you remember - was there some reason we did it this way? Rather than what I've changed it to... it just seems kinda dumb in hindsight, but I might be missing something

Reply:
I don't remember at all